resources/shaders/game.f.glsl

```diff
@@ -45,9 +47,14 @@ void main() {
 
 	float minBrightness = 2.0 / 16.0;
 	if (lightCheck != -1.) {
-		float ambientIntensity = max(max(v_lightValue.x, v_lightValue.y) / 16.0, minBrightness);
-		float diffuseIntensity = max(v_lightValue.x, v_lightValue.y) / 32.0;
+		const float pi = 3.1415927;
+		const float frequency = 512000;
+
+		float time = mod(u_time, 256000);
 
+		float sunlight = clamp(v_lightValue.x * 0.5 * (1 + cos(2 * pi / frequency * time)), 0, 15);
+		float ambientIntensity = max(max(sunlight, v_lightValue.y) / 16.0, minBrightness);
```

Comment:
Please add a line break before this line (which is line 56, ambientIntensity line)